### PR TITLE
Update TAG labels to match README requirements

### PR DESCRIPTION
**Pull request by Riaankl**
Confirmed the `tag:` function work as expected in #3845 

This PR clean up all `cncf_tag` labels in the file

### CODE REVIEW

File: landscape.yml [hunk 1]
```diff
@@ -4229,8 +4219,7 @@ landscape:
               accepted: '2017-09-13'
               incubating: '2017-09-13'
               graduated: '2018-11-28'
-              cncf_tags:
-                - https://github.com/cncf/tag-network
+              tags: tag-network
```
Comment (tegioz) [inline landscape.yml:4222]: This one should be:

```yaml
tag: network
```

File: landscape.yml [hunk 2]
```diff
@@ -4656,8 +4645,7 @@ landscape:
               accepted: '2022-09-30'
               incubating: '2022-09-30'
               graduated: '2023-07-12'
-              cncf_tags:
-                - https://github.com/cncf/tag-network
+              tags: tag-network
```
Comment (tegioz) [inline landscape.yml:4648]: Same as the previous one 🙂

**Review by tegioz — CHANGES_REQUESTED** [review]
Thanks @Riaankl!

**Review by tegioz — APPROVED** [review]
Looks good to me 👍

File: landscape.yml (hunk 1 above)
Comment (Riaankl) [reply landscape.yml]: Good catch. Updated

File: landscape.yml (hunk 2 above)
Comment (Riaankl) [reply landscape.yml]: Good catch. Updated

**Comment by Riaankl** [comment]
@jeefy if you are ok with the changes I think we can merge. Thanks.

**Pull request merged**